client/connection.h

```diff
@@ -11,7 +11,7 @@
 
 #include <mutex>
 
-#include "zmq.hpp"
+#include "../BWEnv/include/zmq.hpp"
```

Comment:
Tbh I would have liked to do the opposite - i.e. for BWEnv to look for `torchcraft/include/zmq.hpp`, but this is outside of the scope of the PR, and it makes sense to standardise this way right now.

Reply:
Not a hard change to make, though zmq.hpp is in /client/ right now, not /include/.

Reply:
Our include paths are all over the place right now, so I wouldn't do this in this PR.